CorePayments/src/main/java/com/paypal/android/corepayments/TrackingEventsAPI.kt

```diff
@@ -45,7 +45,8 @@ internal class TrackingEventsAPI constructor(
             .put(KEY_IS_SIMULATOR, isSimulator)
             .put(KEY_MERCHANT_APP_VERSION, merchantAppVersion)
             .put(KEY_PLATFORM, "Android")
-            .put(KEY_ORDER_ID, event.orderId)
+            .putOpt(KEY_ORDER_ID, event.orderId)
+            .putOpt(KEY_BUTTON_TYPE, event.buttonType)
```

Comment:
Did we update our FPTI tags doc (internal on Excel/Sharepoint) to include this new param as well as the 3 possible string values for each button type?

Reply:
Thanks for the reminder, I just updated it.